src/hud/newhud.asm

```diff
@@ -193,41 +193,151 @@ DrawNewHUD:
         LDA.w #$9DD3 ; What we wrote over
 RTS
 
+; This table works with the table directly below it (HUDChargeIconTable)
+; and needs to be in sync with it. For example, the 3rd index is set to 40
+; so the 3rd set of icons in the HUDChargeIconTable needs to be set to the
+; values for the left and right tile icons that make up "40" on the HUD.
+
+HUDChargeDamageTable:
+dw #$0014 ; 20
+dw #$001E ; 30
+dw #$0028 ; 40
+dw #$0032 ; 50
+dw #$003C ; 60
+dw #$0046 ; 70
+dw #$0050 ; 80
+dw #$005A ; 90
+
+dw #$0064 ; 100
+dw #$00C8 ; 200
+dw #$012C ; 300
+dw #$0190 ; 400
+dw #$01F4 ; 500
+dw #$0258 ; 600
+dw #$0384 ; 900
+
+dw #$0096 ; 150
+dw #$00FA ; 250
+dw #$01C2 ; 450
+dw #$02EE ; 750
+
+dw #$0078 ; 120
+dw #$008C ; 140
+dw #$00B4 ; 180
+
+dw #$00D2 ; 210
+.end
+
+HUDChargeIconTable:
+; |L.Side||R.Side||Table Indexes||Damage Value|
```

Comment:
This comment is a little confusing to me. Just want to confirm the table indexes are on the left side and the damage values are on the right?